docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -401,6 +415,489 @@ docker compose up
 
 You should see the logs reporting a successful connection to the postgres database.
 
+## Providing Updates To the Anchor Platform
```

Comment:
I placed this before `Integration` section so that reader has a background to understand updates in the `Integration` section.
If we place it into the Integration, I think that nested headers will become be too complicated. (For example, menu on the right won't show sub-section of this section anymore and we will have level-5 headers)
Level 4 should be the limit, if there is a level-5 header it means sub-section should be separated into an independent section

Reply:
I think having the `Providing Updates to the Anchor Platform` section as a top-level section is a bit out of place, since we have `Installation`, `Configuration`, and `Integration` currently. I don't think its a problem if our 5th level headers don't show up in the navigation on the right side.